library/entity/networking/src/main/java/org/quiltmc/qsl/entity/networking/api/tracked_data/QuiltTrackedDataHandlerRegistry.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.entity.networking.api.tracked_data;
+
+import net.minecraft.entity.data.TrackedDataHandler;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.quiltmc.qsl.entity.networking.impl.QuiltEntityNetworkingInitializer;
+import org.quiltmc.qsl.registry.api.sync.RegistrySynchronization;
+
+public final class QuiltTrackedDataHandlerRegistry {
+	private static boolean markForSync = true;
```

Comment:
Ok, I'd rather put the implementation in a purely implementation class instead of letting this live in the API class even as private.